### ISSUE TITLE
No license for Saysettha MX

### ISSUE THREAD

**Issue by imnasnainaec** [silnrsi/fonts#13]
Saysettha MX (id `saysetthamx`) is a deprecated font from https://laoscript.net/about/, though I cannot determine from the website what its license was. (It may be deprecated, but license is still listed for other deprecated fonts.)

**Comment by LornaSIL** [comment]
If it's using v3.010 it is under the OFL.

**Comment by jvgaultney** [comment]
It may be reasonable to assume it's OFL, but since it's unclear we can't make a clear statement about it. The license field is optional and should be left blank if 'not known'.

**Comment by LornaSIL** [comment]
I looked inside the font.

**Comment by jvgaultney** [comment]
Were you looking at Saysettha or Saysettha MX?

**Comment by LornaSIL** [comment]
Saysetthe MX v3.010 (that is in the [Keyman repo](https://github.com/keymanapp/keyboards/tree/master/release/shared/fonts/laoo/saysettha)) is under the OFL. I don't know about other versions. It's quite possible the author provided that for Keyman.

**Comment by imnasnainaec** [comment]
@jvgaultney Since the available Saysetthe MX is OFL, is that sufficient?

**Comment by jvgaultney** [comment]
Yes - I've updated to say it's OFL. It still remains deprecated. Users should use Saysettha.
